iotedgehubdev/cli.py

```diff
@@ -101,24 +102,37 @@ def main():
               default=Utils.get_hostname(),
               show_default=True,
               help='GatewayHostName value for the module to connect.')
+@click.option('--iothub-connection-string',
+              '-i',
+              required=False,
+              help='Set the connection string of Azure IoT Hub. Note: Use double quotes when supplying this input.')
```

Comment:
Can we use "Set Azure IoT Hub connection string"?

Reply:
The description currently is aligned with device connection string. If we change this way, how about change the device connection string description to "Set Edge device connection string"?

Reply:
yeah, we can say "Set Azure IoT Edge device connection string"

Reply:
ok